lttng/test.json

```diff
@@ -12,6 +12,7 @@
   "ignoredRIDs":[
     "centos",
     "fedora", // see https://github.com/redhat-developer/dotnet-regular-tests/issues/202
+    "alpine", // lttng tracing disabled due to no support for lttng-ust 2.13
```

Comment:
If there's an issue on github/gitlab/jira/whatever with details, linking to that would be nice too.